src/dmd/hdrgen.d

```diff
@@ -1864,10 +1864,17 @@ public:
                 buf.writestring("cast(");
                 buf.writestring(t.toChars());
                 buf.writeByte(')');
-                if (target.ptrsize == 8)
+                if (target.ptrsize == 4)
+                    goto case Tuns32;
+                else if (target.ptrsize == 8)
                     goto case Tuns64;
                 else
-                    goto case Tuns32;
+                    assert(0);
```

Comment:
This will cause a regression that PR #10051 fixed.

Reply:
I see, that was for the 16 bit code. I submit that 16 bit support should be explicit. I'll make it so.

Reply:
Thanks, though I don't remember why I did not have Tuns16 here unlike in other places.  I'll have to rebuild that compiler configuration again soon.